lib/bolt/config.rb

```diff
@@ -276,6 +276,7 @@ def initialize(project, config_data, overrides = {})
       @config_files = []
 
       default_data = {
+        'apply-settings'      => {},
```

Comment:
It seems weird to set both settings and then force users not to, but on the other hand setting the default once we've resolved which one we're using means we'll have to change it again when we remove the option so...I think this is the best way.